graylog2-web-interface/src/pages/IndexSetPage.tsx

```diff
@@ -39,33 +39,39 @@ import type { Indices } from 'stores/indices/IndicesStore';
 import { IndexerOverviewActions, IndexerOverviewStore } from 'stores/indexers/IndexerOverviewStore';
 import { IndexSetsActions, IndexSetsStore } from 'stores/indices/IndexSetsStore';
 import { IndicesActions, IndicesStore } from 'stores/indices/IndicesStore';
+import useProductName from 'customization/useProductName';
 
 const REFRESH_INTERVAL = 2000;
 
-const ElasticsearchUnavailableInformation = () => (
-  <Row className="content">
-    <Col md={8} mdOffset={2}>
-      <div className="top-margin">
-        <Panel
-          bsStyle="danger"
-          header={
-            <span>
-              <Icon name="warning" /> Indices overview unavailable
-            </span>
-          }>
-          <p>
-            We could not get the indices overview information. This usually means there was a problem connecting to
-            Elasticsearch, and <strong>you should ensure Elasticsearch is up and reachable from Graylog</strong>.
-          </p>
-          <p>
-            Graylog will continue storing your messages in its journal, but you will not be able to search on them until
-            Elasticsearch is reachable again.
-          </p>
-        </Panel>
-      </div>
-    </Col>
-  </Row>
-);
+const ElasticsearchUnavailableInformation = () => {
+  const productName = useProductName();
+
+  return (
+    <Row className="content">
+      <Col md={8} mdOffset={2}>
+        <div className="top-margin">
+          <Panel
+            bsStyle="danger"
+            header={
+              <span>
+                <Icon name="warning" /> Indices overview unavailable
+              </span>
+            }>
+            <p>
+              We could not get the indices overview information. This usually means there was a problem connecting to
+              Elasticsearch, and <strong>you should ensure Elasticsearch is up and reachable from {productName}</strong>
```

Comment:
Maybe we can also replace `Elasticsearch` here? The common name we are using to reference to Elasticsearch/Opensearch/Data Node is "the Indexer".

Reply:
I noticed that too and thought about improving it in all descriptions in a different PR. But in these cases we can also improve it directly.

Reply:
I don't have strong opinion on this and think using a generic name like "the indexer" is fine.